.devcontainer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker.io/jsturtevant/hl-wasm:latest
```

Comment:
will switch to the hyperlight wasm container if that merges: https://github.com/hyperlight-dev/hyperlight-wasm/pull/40